docs/faq.md

```diff
@@ -66,7 +66,7 @@ Regarding Ingress, we'll support:
 
 ### Are other Ingress Controllers supported?
 
-For Ingress objects, ExternalDNS will attempt to discover the target hostname of the relevant Ingress Controller automatically. If you are using an Ingress Controller that is not listed above you may have issues with ExternalDNS not discovering Endpoints and consequently not creating any DNS records. As a workaround, it is possible to force create an Endpoint by manually specifying a target host/IP for the records to be created by setting the annotation `external-dns.alpha.kubernetes.io/target` in the Ingress object.
+For Ingress objects, ExternalDNS will attempt to discover the target hostname of the relevant Ingress Controller automatically. If you are using an Ingress Controller that is not listed above you may have issues with ExternalDNS not discovering Endpoints and consequently not creating any DNS records. As a workaround, it is possible to force create an Endpoint by manually specifying a target host/IP for the records to be created by setting the annotation `external-dns.alpha.kubernetes.io/target` in the Ingress object. Note that services also support this annotation.
```

Comment:
This should be documented where it is describing behavior for Service objects. Someone trying to override the target for an Ingress is unlikely to read this section of the documentation.

Reply:
I agree with that but couldn't find that place. Can you point me to it?

Reply:
Looks like there is no provider agnostic documentation for services. Honestly can't find anything.

Reply:
Yeah, documentation of sources is surprisingly scant. That's a project for another set of PRs. I would personally prefer lack of documentation to misplaced pieces of documentation, but others might disagree.

Reply:
I'll remove the comment. There is one on the code part for now.